packages/server/src/automations/steps/openai.ts

```diff
@@ -0,0 +1,105 @@
+import { Configuration, OpenAIApi } from "openai"
+import {
+  AutomationActionStepId,
+  AutomationStepSchema,
+  AutomationStepInput,
+  AutomationStepType,
+  AutomationIOType,
+} from "@budibase/types"
+import * as automationUtils from "../automationUtils"
+import environment from "../../environment"
+
+enum Model {
+  GPT_35_TURBO = "gpt-3.5-turbo",
+  // will only work with api keys that have access to the GPT4 API
+  // GPT_4 = "gpt-4",
```

Comment:
Should we allow it incase people have access to it? Considering it can't be long until its generally available and we might forget about it.

Reply:
Yep, no reason not to! I was planning on holding off merging this until we can provide GPT-4 in cloud anyway.